drivers/cc110x/gnrc_cc110x/gnrc_netdev_cc110x.c

```diff
@@ -14,13 +14,21 @@
 #include "net/gnrc.h"
 #include "cc110x.h"
 #include "cc110x-netdev.h"
+#ifdef MODULE_GNRC_NETIF2
+#include "net/gnrc/netif2.h"
+#else
 #include "net/gnrc/netdev.h"
+#endif
```

Comment:
Does it still make sense here to check for netif2/netdev?

Reply:
Ooops, that is legacy code I still haven't removed. Fixed!